src/applications/my-education-benefits/containers/CustomReviewDOBField.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { formatReviewDate } from 'platform/forms-system/src/js/helpers';
+
+export default function CustomReviewDOBField({ children, uiSchema }) {
+  return (
+    <div className="review-row">
```

Comment:
The parent of a dt/dd needs to be a `<dl>`

Reply:
Whoops! Not sure how this happened.